float64 aspect_ratio            # Aspect ratio of the velcro strip  
float64 aspect_ratio_threshold  # Threshold +/- the given aspect ratio to still look for object
float64 size                    # expected velcro length size to filter out similar aspect ratio blobs
float64 size_threshold          # Threshold +/- the given size to still look for object
string color                    # color of the mask to use
---
geometry_msgs/Pose centroid_pose 